Makefile

```diff
@@ -9,7 +9,7 @@ export IMAGE_VERSION = v1.16.0
 export SIMPLE_VERSION = $(shell (test "$(shell git describe)" = "$(shell git describe --abbrev=0)" && echo $(shell git describe)) || echo $(shell git describe --abbrev=0)+git)
 export GIT_VERSION = $(shell git describe --dirty --tags --always)
 export GIT_COMMIT = $(shell git rev-parse HEAD)
-export K8S_VERSION = 1.21
+export K8S_VERSION = 1.22
```

Comment:
Based on yesterday's conversation with Joe, we have 2 options:
1. Either leave it as 1.21
2. Or we could update `ENVTEST_K8S_VERSION` to 1.22+ and check if the tests run fine. If so, we can merge both the variables.

I would vote for (2) since we have said that release 1.16 is compatible with k8s 1.22, hence it would be nice to verify and make a patch release stating that. Wdyt @jmrodri ?

Reply:
Changed.